terragrunt/accounts/docs-rs-staging/docs-rs/terragrunt.hcl

```diff
@@ -0,0 +1,23 @@
+terraform {
```

Comment:
Ask in Zulip whether we need different states for the web and the builder (considering they share resources like s3 and RDS).

Reply:
It was agreed in Zulip that this does not need to happen. We should keep them in the same state for simplicity sake.